integration_tests/cairo_zero_hint_tests/is_zero.small.cairo

```diff
@@ -0,0 +1,31 @@
+// Returns 1 if x == 0 (mod secp256k1_prime), and 0 otherwise.
+// Serves as integration test for the following hints : 
+// isZeroNondetCode
+// isZeroPackCode
+// isZeroDivModCode
+
+%builtins range_check
+
+from starkware.cairo.common.cairo_secp.field import is_zero, SumBigInt3
```

Comment:
try with `from starkware.cairo.common.cairo_secp.bigint3 import BigInt3`

Reply:
what do you mean ? is_zero takes a `SumBigInt3` not `BigInt3` . And I tried doing `from starkware.cairo.common.cairo_secp.bigint3 import SumBigInt3` but it gave the same error. It works for me locally but it fails on the github CI.

Reply:
agree with hari, the rust vm does this in cairo files :

```
from starkware.cairo.common.cairo_secp.bigint3 import BigInt3, SumBigInt3
```
maybe try importing both `BigInt3`  and `SumBigInt3` this way?

Reply:
Nope, it still doesn't work.

Reply:
Bumped cairo-lang version.